_docs/best-practices/wdl-best-practices.md

```diff
@@ -0,0 +1,96 @@
+---
+title: WDL Best Practices
+permalink: /docs/publisher-tutorials/wdl-best-practices/
+---
+{% include_relative best-practices-intro.md %}
+
+## Authorship Metadata
+
+> For Dockstore 1.5.0+
+
+{% include_relative authorship-metadata-intro.md %}
+
+*wdl/bamqc.wdl*
+```
+workflow BamQC {
+    String SAMTOOLS
+    File BAMFILE
+
+    call flagstat {
+        input : samtools=SAMTOOLS, bamfile=BAMFILE
+    }
+    call bamqc { 
+        input : samtools=SAMTOOLS, bamfile=BAMFILE, xtra_json=flagstat.flagstat_json
+    }
+    meta {
+        author: "Morgan Taschuk"
+        email: "Morgan.Taschuk@oicr.on.ca"
```

Comment:
Is that a real email? If yes, do you want to post it? Probably OK, but just double checking.

Reply:
Yes, I originally had dummy metadata with a simplistic workflow.  Was told to use a more realistic one.

Reply:
I should probably clarify, I wanted to discourage silly or work inappropriate emails, but that doesn't necessarily mean real emails. 

For example, you could use guest@oicr.on.ca with author `Michael Guest` or something generic sounding.